### ISSUE TITLE
Cannot read property 'Polyline

### ISSUE THREAD

**Issue by BrunerChris** [briannaAndCo/Leaflet.Antimeridian#5]
Running the Wrapped.Polyline example leads to UnCaught TypeError: Cannot read property 'Polyline' of undefined in the console.

**Comment by icarter09** [comment]
@BrunerChris are you putting Leaflet within a script tag on index.html of your application? I experienced the same problem since I was using Leaflet through npm.

**Comment by BrunerChris** [comment]
Yes I am putting Leaflet in a script tag on index. I am also using Leaflet through npm.

**Comment by kjrhody** [comment]
Any updates on this? Having the same issue but with `.Polygon`. I have a Meteor site and am accessing Leaflet through several `script` tags in my `index.html` but they are CDN references and that works fine. Since this does not have a CDN I have to reference the file, and Meteor just can't find it regardless of the path. Not sure if something else is wrong. Any chance you could make this a CDN? Might solve the issues above too, if we could just reference it externally.
